src/app/game-state/log.service.ts

```diff
@@ -1,6 +1,7 @@
 import { Injectable } from '@angular/core';
 import { isEmpty } from 'rxjs';
 
+const LOG_MERGE_INTERVAL_MS = 1000;
```

Comment:
Interval to merge messages. Is best tested by `EVENTS`.

![image](https://user-images.githubusercontent.com/3268868/179387296-85b7f81f-6b0b-4430-b0bf-27e9eaa09579.png)